samples/sampler/stories/qa/Touchable.js

```diff
@@ -1,194 +1,255 @@
-// 			</TouchableDiv>
-// 		)
-// 	)
-// 	.add(
-// 		'onTap when clicked',
-// 		() => (
-// 			<TouchableDiv
-// 				disabled={boolean('disabled', TouchableDiv)}
-// 				noResume={boolean('noResume', TouchableDiv, false)}
-// 				onClick={action('onClick')}
-// 				onDown={action('onDown')}
-// 				onMouseDown={action('onMouseDown')}
-// 				onMouseUp={action('onMouseUp')}
-// 				onTap={action('onTap')}
-// 				onTouchEnd={action('onTouchEnd')}
-// 				onTouchStart={action('onTouchStart')}
-// 				onUp={action('onUp')}
-// 				style={{border: '2px dashed #888', textAlign: 'center'}}
-// 			>
-// 				Click here
-// 			</TouchableDiv>
-// 		)
-// 	);
+import kind from '@enact/core/kind';
+import Button from '@enact/moonstone/Button';
+import {action} from '@enact/storybook-utils/addons/actions';
+import {boolean, number, range} from '@enact/storybook-utils/addons/controls';
+import ri from '@enact/ui/resolution';
+import Touchable from '@enact/ui/Touchable';
+import PropTypes from 'prop-types';
+import {Fragment} from 'react';
+
+const TouchableDiv = Touchable('div');
+
+const onInteractionEnd = () => {
+	const el = document.getElementById('touchRadius');
+	el.style.display = 'none';
+};
+
+const onHoldEnd = (ev) => {
+	onInteractionEnd(ev);
+	return action('onHoldEnd')(ev);
+};
+
+const TouchArea = kind({
+	name: 'TouchArea',
+
+	propTypes: {
+		moveTolerance: PropTypes.number
+	},
+
+	handlers: {
+		onInteractionStart: (ev, {moveTolerance}) => {
+			const el = document.getElementById('touchRadius');
+			const x = ev.clientX || ev.touches && ev.touches[0].clientX;
+			const y = ev.clientY || ev.touches && ev.touches[0].clientY;
+
+			el.style.display = 'block';
+			el.style.left = `${x - moveTolerance}px`;
+			el.style.top = `${y - moveTolerance}px`;
+			return false;
+		}
+	},
+
+	render: ({children, moveTolerance, onInteractionStart, ...rest}) => (<Fragment>
+		<TouchableDiv
+			{...rest}
+			onHoldEnd={onHoldEnd}
+			onMouseDown={onInteractionStart}
+			onMouseUp={onInteractionEnd}
+			onTouchEnd={onInteractionEnd}
+			onTouchStart={onInteractionStart}
+		>
+			{children}
+		</TouchableDiv>
+		<div
+			id="touchRadius"
+			style={{
+				display: 'none',
+				position: 'fixed',
+				height: (moveTolerance * 2) + 'px',
+				width: (moveTolerance * 2) + 'px',
+				borderRadius: '999px',
+				border: '1px solid orange',
+				backgroundColor: 'rgba(255, 180, 0, 0.3)',
+				pointerEvents: 'none',
+				touchAction: 'none'
+			}}
+		/>
+	</Fragment>)
+});
+
+export default {
+	title: 'Moonstone/Touchable',
+	component: 'Touchable'
+};
+
+export const WithDefaultHoldEvents = (args) => (
+	<Button
+		disabled={args['disabled']}
+		onHold={action('onHold')}
+		onHoldEnd={action('onHoldEnd')}
+		onHoldStart={action('onHoldStart')}
+	>
+		Touchable
+	</Button>
+);
+
+boolean('disabled', WithDefaultHoldEvents, Button);
+
+WithDefaultHoldEvents.storyName = 'with default hold events';
+
+export const WithCustomLongPress = (args) => (
```

Comment:
This story seems to do the same thing as the one below it. If that is the case please remove it. It does not appear to exist with knobs.